packages/pages/src/upgrade/migrateConfig.test.ts

```diff
@@ -0,0 +1,33 @@
+import { describe, it, expect, vi } from "vitest";
+import { formatSiteStream } from "./migrateConfig.js";
+
+const siteStreamPath = "foo/bar";
+
+describe("formatSiteStream", () => {
+  it("errors and exits when there are multiple entityIds", () => {
+    const testJson = { filter: { entityIds: ["1234", "123"] } };
+    const mockExit = vi
+      .spyOn(process, "exit")
+      .mockImplementation(() => undefined as never);
+    formatSiteStream(testJson, siteStreamPath);
+    expect(mockExit).toHaveBeenCalledWith(1);
```

Comment:
Do you want toHaveBeenCalledTimes here? You're not passing a particular function param was called

Reply:
Oh I meant toHaveBeenCalledWith(1) as it calls an exit 1.